frontend/app/quest/Quest_Activity.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+    <!-- to do:  update central stylesheet to this style sheet code.-->
+    
+<html ng-app="quest_builder">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Hello">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="shortcut icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <title>PlanetLAB> Quest Builder</title>
+        <link rel="stylesheet" href="css/style.css" type="type/css" />
+        </head>
+
+    <body>
+    <div ng-view></div>
+        <img src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games">
+        <ol class="controlNav" >
+            <li class="questControlNav"> 
+                <a  href="Quest_Basic.html">Basic Info</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_Activity.html">Activity Details</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_for teachers.html">For Teachers</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_Review.html">Review</a>
+            </li>
+        </ol>
+ <h1>Quest: Activity Details</h1>       
+<form>
+<fieldset>
+	<section class="row">
+Duration: <input type="text" Duration="Duration" > <!--format HH:mm:ss-->
+	</section>
+
+<section class="row">
+Material &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+QTY &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+Cost &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <br><br> <!--adding more as needed?? JScript??-->
+<input type="text" Material="material" >      
+<input type="text" QTY="QTY" >
+<input type="text" Cost="Cost" ><br>
+	</section>
+<br><br>
+<section class="row">
+Enquiry Questions:<br><br>
+<input type="text" Enquiry="Enquiry" >
+
+	</section>
+
+<section class="row">Page Break</section>
+<section class="row"> QUIZ
+<!-- MCQ unkown questions needs same add more JScript??-->
+<input type="text" Quiz="Quiz" >
+<input type="text" answer="answer" ><!-- radio for right answer-->
+
+
+</section>
+
+
+</fieldset>
+</form>
+
+<script src="Banner.js"></script>
+
+
+    <nav>
+
+      <ul>
+          <button class="navbtn" id="btn1">Back</a></button>
+          <button class="navbtn" id="btn2">Save/Exit</a></button>
+          <button class="navbtn" id="btn3">Next</a></button>
+      </ul>
+    </nav>
+
+    <h3 id="ars" > Confirm Action </h3>
+ 
+    
+    <script type="text/javascript" src="jquery.js"></script>
+<script>
+
+  $(function () {
```

Comment:
Scripts should be in an external file... but we shouldn't be doing any jQuery anyway. This should all be done through the ui-router